resources/js/app.ts

```diff
@@ -18,7 +18,9 @@ import Layout from './Pages/Layout.vue';
         const i18nPlugin = createI18n({
             locale: locale,
             messages: i18nMessages,
-            wikilinks: true
+            wikilinks: true,
+            globalInjection: true,
+            legacy: false
```

Comment:
There are actually not from the vue-banana-i18n library, we do not need them anymore.